lib/normalizer.c

```diff
@@ -2091,6 +2561,7 @@ grn_nfkc_normalize_unify(grn_ctx *ctx,
         data->options->unify_katakana_di_sound ||
         data->options->unify_katakana_gu_small_sounds ||
         data->options->unify_katakana_trailing_o ||
+        data->options->unify_kana_prolonged_sound_mark ||
```

Comment:
Does this work with `unify_katakana_trailing_o`?

Reply:
I will add test and fix implementation if need.

Reply:
That didn't work.
I have fixed it and add tests.